src/shared-components/dropdown/test.tsx

```diff
@@ -16,9 +16,10 @@ const options = [
 describe('<Dropdown />', () => {
   describe('on touch screen', () => {
     it('renders <MobileDropdown />', () => {
-      global.document.documentElement.ontouchstart = () => {};
+      // @ts-ignore TODO: Add global interface to TS definitions https://stackoverflow.com/questions/40743131/how-to-prevent-property-does-not-exist-on-type-global-with-jsdom-and-t
+      global.document.documentElement.ontouchstart = () => undefined;
```

Comment:
It looks like we can use `window` instead of `global` and the test suite still passes, without the need for `// @ts-ignore` directives. (I couldn't tell you why this is, though. Maybe enzyme v3 changed `window` behavior.)